webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,46 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+
+export function notify(message, type, sticky) {
```

Comment:
The Toast component accepts a link property which displays a link inline after the message.
I agree that the toast and notification drawer should support the same interface, including an action dropdown menu. Toast dropdown not yet implemented.

There is no support for html in the toast message. It is possible to implement via children (transclude for you Walden) but I would prefer not. For more specific formatting I prefer adding properties to the toast component and using a structured format instead of free form html.